src/test/groovy/org/gradle/profiler/AsyncProfilerIntegrationTest.groovy

```diff
@@ -4,102 +4,141 @@ package org.gradle.profiler
 import spock.lang.Requires
 
 class AsyncProfilerIntegrationTest extends AbstractProfilerIntegrationTest {
-    @Requires({ OperatingSystem.isMacOS() })
-    def "profiles build using async-profiler with tooling API and warm daemon"() {
+    @Requires({ !OperatingSystem.isWindows() })
+    def "profiles build CPU usage using async-profiler with tooling API and warm daemon"() {
         given:
         instrumentedBuildScript()
 
         when:
-        new Main().run("--project-dir", projectDir.absolutePath, "--output-dir", outputDir.absolutePath, "--gradle-version", minimalSupportedGradleVersion, "--profile", "async-profiler",  "assemble")
+        new Main().run("--project-dir", projectDir.absolutePath, "--output-dir", outputDir.absolutePath, "--gradle-version", latestSupportedGradleVersion, "--profile", "async-profiler", "assemble")
```

Comment:
❓ shouldn't we test both `minimalSupportedGradleVersion` and `latestSupportedGradleVersion`?

Reply:
yes, probably. I figured other tests took care of covering the actual Gradle invocation stuff for all of the versions, and this test can assume that it works fine. I changed it to the latest version because this is more likely to be used with the async-profiler integration (rather than Gradle 3.3, which is the minimal version) and also so I didn't need to figure out how to run the tests with Java 8 (Gradle 3.3 doesn't work on Java 11).